clevis/Sanity/kernel-keyring-support/Makefile

```diff
@@ -0,0 +1,64 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Makefile of /CoreOS/clevis/Sanity/kernel-keyring-support
+#   Description: Test clevis to provide a parameter for the key description of a key in the kernel keyring
```

Comment:
I am not sure Description is correct.
This new parameter (-e) allows specifying a LUKS2 token id so that password is read from it. I would add something like:
Description: Test clevis to check parameter that allows reading a LUKS2 token id to avoid password prompt for existing LUKS2 password and read it from the key description associated to that token id

Reply:
Provided description according to your proposal.